.github/workflow_scripts/env_setup.sh

```diff
@@ -18,6 +18,7 @@ function setup_build_contrib_env {
 
 function setup_benchmark_env {
     pip install -U autogluon.bench
+    pip install pyarrow
```

Comment:
Is this a temporary fix? If yes, please insert a TODO note for its subsequent removal.

Reply:
This is a temporary fix, version 0.4.3 of AG bench required this package in order for evaluation scripts to work.
But it was released without this package being added
In order to avoid another release process of AG Bench, I added it here.
Inserted TODO